RecoLocalCalo/EcalRecProducers/python/ecalMultiFitUncalibRecHit_cff.py

```diff
@@ -64,3 +67,40 @@
   # ECAL multifit running on CPU, or convert the uncalibrated rechits from SoA to legacy format
   ecalMultiFitUncalibRecHit,
 ))
+
+# modifications for alpaka
+from Configuration.ProcessModifiers.alpaka_cff import alpaka
+
+# ECAL conditions used by the multifit running on the accelerator
+from RecoLocalCalo.EcalRecProducers.ecalMultifitConditionsHostESProducer_cfi import ecalMultifitConditionsHostESProducer
+from RecoLocalCalo.EcalRecProducers.ecalMultifitParametersHostESProducer_cfi import ecalMultifitParametersHostESProducer
+
+ecalMultifitParametersSource = cms.ESSource("EmptyESSource",
+    recordName = cms.string('EcalMultifitParametersRcd'),
+    iovIsRunNotTime = cms.bool(True),
+    firstValid = cms.vuint32(1)
+)
+
+# ECAL multifit running on the accelerator
+from RecoLocalCalo.EcalRecProducers.ecalUncalibRecHitProducerPortable_cfi import ecalUncalibRecHitProducerPortable as _ecalUncalibRecHitProducerPortable
+ecalMultiFitUncalibRecHitPortable = _ecalUncalibRecHitProducerPortable.clone(
+  digisLabelEB = 'ecalDigisPortable:ebDigis',
+  digisLabelEE = 'ecalDigisPortable:eeDigis'
+)
+
+# replace the SwitchProducerCUDA branches with the module to convert the uncalibrated rechits from SoA to legacy format
+from RecoLocalCalo.EcalRecProducers.ecalUncalibRecHitConvertPortable2CPUFormat_cfi import ecalUncalibRecHitConvertPortable2CPUFormat as _ecalUncalibRecHitConvertPortable2CPUFormat
+alpaka.toModify(ecalMultiFitUncalibRecHit,
+    cpu = _ecalUncalibRecHitConvertPortable2CPUFormat.clone(),
+    cuda = _ecalUncalibRecHitConvertPortable2CPUFormat.clone()
```

Comment:
Same here: does it work to skip the `cuda` branch ?

Reply:
According to @makortel no.

Reply:
According to my test yes ...

Reply:
OK I'll remove it then.